scripts/cibuild

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -e
+
+if [[ -n "${GT_SITE_DEBUG}" ]]; then
+    set -x
+fi
+
+if [[ -n "${GIT_COMMIT}" ]]; then
+    GIT_COMMIT="${GIT_COMMIT:0:7}"
+else
+    GIT_COMMIT="$(git rev-parse --short HEAD)"
+fi
+
+function usage() {
+    echo -n \
+"Usage: $(basename "$0")
+Build application for staging or release.
+"
+}
+
+if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
+    if [ "${1:-}" = "--help" ]; then
+        usage
+    else
+
+    make clean
+    ./scripts/test
+    make assembly
+    make assets
+    GIT_COMMIT="${GIT_COMMIT}" make build
```

Comment:
The `GIT_COMMIT` environment override is coded into the Makefile. Is it perferred to use `-e`?